teams/vsa/teams/qa/vsa-qa-process.md

```diff
@@ -0,0 +1,55 @@
+# Process
+
+\[Based on [template process](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Practice%20Areas/QA/QA_Testing_Matrix_Template.xlsx) by VSP Product Support team Sr. QA Specialist Peter Hill]
+
+This VSA-QA Process is currently a subset of the base-process as defined by VSP Product Support, and will grow over time to include more from this base as staffing expands.
+
+![VSA-QA Process][qaprocess]
+---
+
+1. Understand product/feature updates: discover how to test the changes being tackled in your sprint. \[As stated above, VSA-QA currently handles only manual UI testing on Staging.]
```

Comment:
Add new Step 1: Engineering team create Test Plan ticket & assign to VSA-QA resource (currently @tlei123).

Reply:
Done & pushed.